restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/domain/SupportedAsrLanguages.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * TeleStax, Open Source Cloud Communications
+ * Copyright 2011-2014, Telestax Inc and individual contributors
+ * by the @authors tag.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation; either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package org.restcomm.connect.interpreter.rcml.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Library class with all supported languages.
+ *
+ * @author Dmitriy Nadolenko
+ */
+public class SupportedAsrLanguages {
```

Comment:
@DmytroDev I believe we should have the languages at the restcomm.xml and load them with configuration, similar to drivers. This way we add add/remove or enable/disable languages without changing source code. Wdyt?